FAQ.md

```diff
@@ -30,10 +31,32 @@ are not supported by RuneLite.
 If you are unsure if your bug is caused by a third party plugin or not, run the client in *safe mode* and try to reproduce the issue. Safe mode disables
 all plugin hub plugins, and also GPU mode.
 
-You can run in safe mode by either running the `RuneLite (safe mode)` shortcut on Windows (available in launcher 2.6.3), or passing `--safe-mode` to the RuneLite executable.
+You can run in safe mode by either running the `RuneLite (safe mode)` shortcut on Windows (available in launcher 2.6.3), or passing `--safe-mode` to the RuneLite executable. For more information, see [below](#how-do-i-run-runelite-in-safe-mode).
```

Comment:
I was stumped on where to put this launcher-updating-info, but now that Adam has suggested bumping safe-mode higher in the file, it actually fits.
```suggestion
You can run in safe mode by either running the `RuneLite (safe mode)` shortcut on Windows (available in launcher 2.6.3), or passing `--safe-mode` to the RuneLite executable. To update your launcher please visit the [RuneLite website](https://runelite.net/) and select to download the setup file for your Operating System then install the launcher using that.
```

Reply:
I'm not sure this makes a lot of sense. It would probably be better to move all Safe Mode-related info to the one FAQ question. Otherwise the info will be spread out and hard to find for readers.

Reply:
> Otherwise the info will be spread out and hard to find for readers.

Did you not notice that you've moved everything from this PR right next to eachother?
![image](https://github.com/runelite/wiki/assets/41973452/0ec8314b-12aa-4596-9aad-28be67cd2d3b)

Reply:
Right, that's correct. It is the no. 2 question in the FAQ as recommended by @Adam- . However, you are suggesting that the Safe Mode info be moved from the second question (Safe Mode) to the first question (Common Bugs).

Reply:
That's not what I'm suggesting, we are both editing a section that's already in `(Common Bugs)`, I just want the update information to be somewhere useful.

Reply:
OK, gotcha. I think it might be best to just get this PR merged, then we can improve on it later.